locales-src/newtab.ftl

```diff
@@ -131,6 +131,13 @@ newtab-section-menu-move-up = Move Up
 newtab-section-menu-move-down = Move Down
 newtab-section-menu-privacy-notice = Privacy Notice
 
+## Section aria-labels
+
+newtab-section-collapse-section-label =
+    .aria-label = Collapse Section
+newtab-section-expand-section-label =
+    .aria-label = Expand Section
+
```

Comment:
Sorry missed Including @flodolo for strings review. @Mardak do you know why this PR didn't add @flodolo for review by default for newtab.ftl?

Reply:
Only strings.properties is listed:
https://github.com/mozilla/activity-stream/blob/68a8907fd1f8c0f27b3983d0ff1b969859e43392/CODEOWNERS#L1-L2

Probably should just add the whole directory if we can.

Reply:
Created https://github.com/mozilla/activity-stream/pull/5202